cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/cluster/KafkaClusterOperations.java

```diff
@@ -137,21 +137,21 @@ public void create(String namespace, String name, Handler<AsyncResult<Void>> han
 
             CompositeFuture
                 .join(createResult)
-                .compose(res -> statefulSetOperations.waitUntilReady(namespace, zk.getName(), 60, TimeUnit.SECONDS))
+                .compose(res -> statefulSetOperations.waitUntilReady(namespace, zk.getName(), 1, 60, TimeUnit.SECONDS))
```

Comment:
I think our life would be easier if we used the same units everywhere ... like milliseconds. WDYT?

(Sorry, I know I'm the one who started with seconds here and milliseconds inside the `waitUntilReady` method :-()